projects/bp-gallery/src/views/shared/picture-info/TagSelectionField.tsx

```diff
@@ -25,17 +29,20 @@ const TagSelectionField = <T extends TagFields>({
   createMutation,
   nonVerifyable = false,
   noContentText,
+  type,
 }: {
   tags: T[];
   allTags: T[];
   onChange?: (tags: T[]) => void;
   createMutation?: (attr: any) => Promise<any>;
   nonVerifyable?: boolean;
   noContentText: string;
+  type: TagType;
 }) => {
   const { role } = useAuth();
   const { t } = useTranslation();
   const openAlert = useContext(AlertContext);
+  const history: History = useHistory();
```

Comment:
Please remove this unused line (and also the imports for `useHistory` and the `History` type) ^^